plugins/woocommerce/tests/php/src/Internal/Caches/EntityVersionsCacheTest.php

```
@@ -0,0 +1,507 @@
+	 * @testdox get_entity_version refreshes the TTL of the existing entity version.
+	 */
+	public function test_get_entity_version_refreshes_ttl() {
+		// Pre-populate cache with a known version.
+
+		$expected_version               = 'existing-version-uuid';
+		$cache_key                      = 'wc_entity_version_custom_entity_789';
+		$this->sut->cache[ $cache_key ] = $expected_version;
+
+		// Track set_cached calls by monitoring the cache.
+
+		$initial_cache_state = $this->sut->cache;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$initial_cache_state'. (undefined)

(UnusedLocalVariable)